android/src/main/java/com/mrousavy/blurhash/BlurhashViewManager.kt

```diff
@@ -23,7 +23,7 @@ internal class LoadStartEvent(
   override fun getEventData(): WritableMap = Arguments.createMap()
 
   companion object {
-    const val EVENT_NAME = "topBlurhashLoadStart"
+    const val EVENT_NAME = "topLoadStart"
```

Comment:
Can `topLoadStart` theoretically collide with other components that emit events under the same name, like e.g. an `Image` component?

Reply:
ok cool lgtm then haha thx